### DISCUSSION TITLE
Beginner questions: Decap CMS and Hugo (Themes)

### DISCUSSION THREAD

**Discussion by rolffresh** [decaporg/decap-cms#7150] [Q&A] [answered]
Hi, 

can I use Decap CMS with any Hugo Theme?

It the structure between Hugo sites always similar, so Decap CMS knows where the pages/posts are stored etc.?

What I accomplished so far is setting up Hugo and deploying an example site to Netlify via GitHub.
Now I disocvered DecapCMS and would like to set it up to work with my site.
Would following the "Add To Your Site" guide exactly accomplish that?

I also ran the quickstart on Netlify, and DecapCMS is exactly the missing link I was looking for.

Thanks for tips.

**Comment by Armoghan-ul-Mohmin** [comment, marked as answer]
Yes, Decap CMS can be used with any Hugo theme. It understands the typical file structure of Hugo sites, making it compatible with where your pages and posts are stored. Ensure Decap CMS is configured to point to your content directory and define the content types you want to manage, and it should work well with your Hugo site

**Reply by rolffresh** [reply]
Thank you. Will try it out!
